### ISSUE TITLE
cpSegmentShapeSetNeighbors support in CCPhysics

### ISSUE THREAD

**Issue by tanis2000** [cocos2d/cocos2d-objc#541]
I noticed that in `bodyWithPolylineFromPoints` there's this comment:

```
// TODO Broken. Values may be wrong after applying a transform in onEnter.
//cpSegmentShapeSetNeighbors(shape.shape.shape, points[(i - 1 + count)%count], points[(i + 2)%count]);
```

I'm pretty sure that support for cpSegmentShapeSetNeighbors could solve most of my problems with squared corners and moving objects that get stuck on them. Is there any plan to add support for it?

**Comment by slembcke** [comment]
Yes. I think I have a solution that will work for it too, though it's not very high priority at the moment.

**Comment by tanis2000** [comment]
@slembcke for the time being I've uncommented that line of code and it really solved pretty much all the problems I had with squared corners and moving enemies. I suppose that it works for me as I'm not applying any kind of transform to those bodies as they're all static bodies created after loading a tile map. 

I just need to remember to uncomment that line every time I update cocos2d-iphone from GitHub though :)

**Comment by slembcke** [comment]
This is included in #479.
